### ISSUE TITLE
Accessibility: high contrast mode does not show the icon in the disabled splitbuttons Examples

### ISSUE THREAD

**Issue by kenotron** [OfficeDev/office-ui-fabric-react#10496]
### Environment Information

- **Package version(s)**: 7.x

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Splitting out #6356 to smaller bugs to be fixed. SplitButton examples in high-contrast mode does not show the disabled chevron down buttons

### Repro

1. Turn on highcontrast mode, open Edge
2. Open 'https://developer.microsoft.com/en-us/fabric#/components/button' in Edge.   
3. Look for the disabled SplitButton examples

### Actual

![image](https://user-images.githubusercontent.com/34725/65095682-13523100-d977-11e9-8bea-13600a40607a.png)

### Expected

The ChevronDown icons should be visible in high contrast mode

**Comment by kenotron** [comment]
This is no longer repro'ing for Fabric 7:

![image](https://user-images.githubusercontent.com/34725/65171919-d894db00-da00-11e9-91ab-8050bf8701c5.png)
